### PR TITLE
wip: add tests,add new target parameter,make it major by default

### PR DESCRIPTION
**Pull request by fahric**
wip for working version difference between from and to versions

#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: src/lib/checkTargetMatchToPR.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  }
+  const titleSplitByFrom = prTitle.split('from')
+  if (titleSplitByFrom.length === 2) {
+    const versionsArray = titleSplitByFrom[1].split('to')
+    if(versionsArray.length === 2) {
+      const [fromVersion, toVersion] = versionsArray
+      const fromVersionArray = fromVersion.trim().split('.')
+      const toVersionArray = toVersion.trim().split('.')
+      if(fromVersionArray.length === 3 && toVersionArray === 3){
+        // TODO: we know know that we have version numbers so implement the logic
```
Comment (mcollina) [inline src/lib/checkTargetMatchToPR.js:16]: what is this? Could it be resolved?

**Review by mcollina — COMMENTED** [review]
Good work!

File: src/lib/checkTargetMatchToPR.js (hunk 1 above)
Comment (fahric) [reply src/lib/checkTargetMatchToPR.js:16]: sorry Matteo, it should have been draft PR

**Comment by fahric** [comment]
should have opened a draft PR

**Pull request closed without merging**